docs/adr/0009-enable-redis-on-productiondata.md

```diff
@@ -0,0 +1,25 @@
+# 9. Enable Redis on productiondata
+
+Date: 2024-07-17
+
+## Status
+
+Accepted
+
+## Context
+
+In preparation for the next academic year we need to test the changes that we've made on the `productiondata`
+
+Generally the worker is off as there is no need for it to be running all the time.
+
+It is also off be default to prevent any unintentional mishaps.
+
+## Decision
+
+We've enabled Redis for `productiondata` so as to for the Sidekiq workers to be functional
+
+## Consequences
+
+1. Extra cost as an instance of Redis is active
+2. We have to be mindful of what actions we perform in the application and what jobs those may trigger
+3. After we have completed our testing the Redis instance should be switched off
```

Comment:
There are Redis instances running for productiondata. We just have 0 worker processes, and changing replicas to 1 in worker_apps will start a worker, which will connect to the existing redis. Setting back to 0 will then stop the worker, but leave redis running (which is expected).

Reply:
Correct. My bad, rephrased